app/controllers/carts_controller.rb

```diff
@@ -1,4 +1,14 @@
 class CartsController < ApplicationController
+  def checkout_cart
+    if session[:user_id].nil?
+      flash[:alert] = 'You must login to checkout'
+      redirect_to showcart_path
+    else
+      require "pry"; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.